app/src/main/java/io/novafoundation/nova/app/root/presentation/deepLinks/handlers/DAppDeepLinkHandler.kt

```diff
@@ -27,12 +30,13 @@ class DAppDeepLinkHandler(
     }
 
     override suspend fun handleDeepLink(data: Uri) {
-        // TODO: check that user has accounts here
-        val url = data.getDappUrl() ?: return
-        val normalizedUrl = Urls.normalizeUrl(url)
+        if (!accountRepository.hasMetaAccounts()) return
```

Comment:
This protects us from opening browser after pin code (if it's cold start and user create account first time).
But maybe we need to open it after creating first account. In this case we can remove this line

Reply:
Since we perform ` automaticInteractionGate.awaitInteractionAllowed()`, the flow will only proceed after main screen opens. e.g. when user have some account so I do not see why this check is needed